pkg/kubectl/cmd/drain.go

```diff
@@ -266,40 +267,36 @@ func (o *DrainOptions) deleteOrEvictPodsSimple() error {
 	return err
 }
 
-func (o *DrainOptions) getController(sr *api.SerializedReference) (interface{}, error) {
-	switch sr.Reference.Kind {
+func (o *DrainOptions) getController(pod *api.Pod, controllerRef *metav1.OwnerReference) (interface{}, error) {
```

Comment:
there is already a sanity check in `getPodController()` function.
interesting. if we remove `getController, `getPodController()` will become an inline function.